src/Console/Command/CommandHelp.php

```diff
@@ -286,6 +286,8 @@ public static function getDisplayableAllowedValues(FixerOption $option)
                 return !is_callable($value);
             });
 
+            sort($allowed);
```

Comment:
I'm not sure about automatically sorting allowed values as well. They may have been set in a more logical order in the configuration definition. We should make sure they are already in the desired order there instead.

For example: the values of the `separate` option for `header_comment` fixer are in a weird order now, I would at least make `none` the very last item.